lib/pbench/server/api/__init__.py

```diff
@@ -49,139 +49,138 @@ def register_endpoints(api, app, config):
     to make the APIs active."""
 
     base_uri = config.rest_uri
-    logger = app.logger
 
     # Init the the authentication module
     token_auth = Auth()
-    Auth.set_logger(logger)
+    Auth.set_logger(current_app.logger)
     Auth.set_oidc_client(server_config=config)
 
-    logger.info("Registering service endpoints with base URI {}", base_uri)
+    app.logger.info("Registering service endpoints with base URI {}", base_uri)
```

Comment:
What is the difference between `current_app` and `app` in this context?  Aren't they referring to the same thing?  (I hope so, since they are both being used to refer to the logger....)

I believe that using `current_app` has the liability that it only works when there is context on the stack, which leads to the strange idiom at line 215.  If we use `app` here, instead, then I think we can revert the change made there.